src/utils/parsePhone.ts

```diff
@@ -0,0 +1,19 @@
+
+
+//Takes in a string and returns either the forst valid phone number or null
+
+export function parsePhoneNumber(potentialNumber: string): string | null {
+    let phoneRegex = /[+]*[(]{0,1}[0-9]{1,4}[)]{0,1}[-\s\.0-9]*/
+    let eightHundredRegex = /0800\s[A-Z]+/
+    //Does a standard phone regex, but that will also just return the 0800 from a number like 0800 ORANGA
+    //So I check its length and do an 0800 regex as well
```

Comment:
Small ask to improve the comment, remember other people need to read this in the context of what the code does, not what you ("so _I_ check") do. Thanks!